agb-fixnum/src/lib.rs

```diff
@@ -570,7 +570,8 @@ impl<I: FixedWidthUnsignedInteger, const N: usize> Debug for Num<I, N> {
 }
 
 /// A vector of two points: (x, y) represened by integers or fixed point numbers
-#[derive(Clone, Copy, PartialEq, Eq, Debug)]
+#[derive(Clone, Copy, PartialEq, Eq, Debug, Default)]
+#[repr(C)]
```

Comment:
rather not commit to a representation

Reply:
I'm tempted to leave that in until we add your affine matrix fun PR?